Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -395,6 +395,7 @@ def _getSeedingLayers(seedProducers):
     trackingParticleNumberOfLayersProducer
 )
 eras.fastSim.toModify(tracksValidationTruth, lambda x: x.remove(tpClusterProducer))
+eras.trackingPhase2PU140.toModify(tpClusterProducer, pixelSimLinkSrc = cms.InputTag("simSiPixelDigis", "Pixel"))
```

Comment:
The `tpClusterProducer` should be customized where it was defined.

Add `SimTracker/TrackerHitAssociation/python/tpClusterProducer_cff.py` with
```
from SimTracker.TrackerHitAssociation.tpClusterProducer_cff import *
eras.trackingPhase2PU140.toModify(tpClusterProducer, pixelSimLinkSrc = "simSiPixelDigis:Pixel")
```
and replace all imports of `tpClusterProducer_cfi` to `_cff`.

OR

Modify ClusterTPAssociation to generate e.g. `tpClusterProducerDefault_cfi.py`
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/SimTracker/TrackerHitAssociation/plugins/ClusterTPAssociationProducer.cc#L83
and then add Add `SimTracker/TrackerHitAssociation/python/tpClusterProducer_cfi.py` with
```
from SimTracker.TrackerHitAssociation.tpClusterProducerDefault_cfi import tpClusterProducerDefault as _tpClusterProducerDefault
tpClusterProducer = _tpClusterProducerDefault.clone()
eras.trackingPhase2PU140.toModify(tpClusterProducer, pixelSimLinkSrc = "simSiPixelDigis:Pixel")
```
In this case all existing imports of `tpClusterProducer_cfi` continue to work.

Which to choose is mostly matter of taste. Latter is a bit simpler since all existing imports continue to work.